CI/windows/build_ffmpeg.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+################################################################################
+# Windows FFmpeg cross-compile build script
+################################################################################
+#
+# This script file can be included in build scripts for Windows or run directly
+#
+################################################################################
+
+# Halt on errors
+set -eE
+
+_patch_product() {
+    cd "${PRODUCT_FOLDER}"
+
+    set +eE
+    check_git
+    set -eE
+
+    step "Apply patches..."
+    apply_patch "${CHECKOUT_DIR}/CI/windows/patches/ffmpeg/ffmpeg_flvdec.patch" "2e73d9296c3190a9e395c1e0dfe98e9b12df40e5960dc27d5cd19c7e8d8695ab"
+    git add .
+    git commit -m "Fix decoding of certain malformed FLV files"
+
+    git cherry-pick 1f7b527194a2a10c334b0ff66ec0a72f4fe65e08 \
+        f9d6addd60b3f9ac87388fe4ae0dc217235af81d \
+        79d907774d59119dcfd1c04dae97b52890aec3ec \
+        8d823e6005febef23ca10ccd9d8725e708167aeb \
+        952fd0c768747a0f910ce8b689fd23d7c67a51f8 \
+        d7e2a2bb35e394287b3e3dc27744830bf0b7ca99 \
+        3def315c5c3baa26c4f6b7ac4622aa8a3bfb46f8 \
+        f8990c5f414d4575415e2a3981c3b142222ca3d4 \
+        fee4cafbf52f81ffd6ad7ed4fd0a8096f8791886 \
+        b96bc946f219fbd28cffc1efea78fd42f34148ec \
+        006744bdbd83d98bc71cb041d9551bf6a64b45a2 \
+        aab9133d919bec4af54a06216d8629ebe4fb8f74 \
+        c112fae6603f8be33cf1ee2ae390ec939812f473 \
+        86a7b77b60488758e0c080882899c32c4a5ee017 \
+        7cc7680a802c1eee9e334a0653f2347e9c0922a4 \
+        449e984192d94ac40713e9217871c884657dc79d \
+        290a35aefed250a797449c34d2f9e5af0c4e006a \
+        6e95ce8cc9ae30e0e617e96e8d7e46a696b8965e \
+        e9b35a249d224b2a93ffe45a1ffb7448972b83f3 \
+        7c59e1b0f285cd7c7b35fcd71f49c5fd52cf9315 \
+        86f5fd471d35423e3bd5c9d2bd0076b14124faee \
+        fb0304fcc9f79a4c9cbdf347f20f484529f169ba
```

Comment:
It would be nice if these were commented with the commit title so its easy to see what they are.

Reply:
I'll see if I can do that quickly.

Reply:
A comment block with the output of `git log --oneline` that matches this commit order has been added.